testsuite/MDAnalysisTests/coordinates/test_timestep_api.py

```diff
@@ -71,162 +92,34 @@ class BaseTimestepInterfaceTest(TestCase):
-    def setUp(self):
-        u = self.u = mda.Universe(DMS)
-        self.ts = u.trajectory.ts
-
-
-class TestGMS(BaseTimestepInterfaceTest):
-    __test__ = True
-    def setUp(self):
-        u = self.u = mda.Universe(GMS_ASYMOPT, GMS_ASYMOPT,
-                                  format='GMS', topology_format='GMS')
-        self.ts = u.trajectory.ts
-
-
-class TestGRO(BaseTimestepInterfaceTest):
-    __test__ = True
-    def setUp(self):
-        u = self.u = mda.Universe(GRO)
-        self.ts = u.trajectory.ts
-
-
-class TestINPCRD(BaseTimestepInterfaceTest):
-    __test__ = True
-    def setUp(self):
-        u = self.u = mda.Universe(XYZ_five, INPCRD)
-        self.ts = u.trajectory.ts
-
-
-class TestLAMMPS(BaseTimestepInterfaceTest):
-    __test__ = True
+    @pytest.fixture()
+    def universe(self, topology, trajectory, trajectory_format, topology_format):
+        if trajectory_format is not None and topology_format is not None:
+            return mda.Universe(topology, trajectory, format=trajectory_format,
+                                topology_format=topology_format)
 
-    def setUp(self):
-        u = self.u = mda.Universe(LAMMPSdata)
-        self.ts = u.trajectory.ts
+        if trajectory is not None:
+            return mda.Universe(topology, trajectory)
+        else:
+            return mda.Universe(topology, format=trajectory_format)
 
+    def test_frame(self, universe):
+        assert_equal(universe.trajectory.ts.frame, 0)
 
-class TestLAMMPSDCD(BaseTimestepInterfaceTest):
-    __test__ = True
+    def test_dt(self, universe):
+        assert_equal(universe.trajectory.dt, universe.trajectory.ts.dt)
 
-    def setUp(self):
-        u = self.u = mda.Universe(LAMMPSdata2, LAMMPSdcd2,
-                                  format='LAMMPS', topology_format='DATA',
-                                  timeunit='fs')
-        self.ts = u.trajectory.ts
 
+class TestNCDFBaseTimestepInterface(object):
 
-class TestMOL2(BaseTimestepInterfaceTest):
-    __test__ = True
-
-    def setUp(self):
-        u = self.u = mda.Universe(mol2_molecules)
-        self.ts = u.trajectory.ts
-
-
-class TestPDB(BaseTimestepInterfaceTest):
-    __test__ = True
-
-    def setUp(self):
-        u = self.u = mda.Universe(PDB_small)
-        self.ts = u.trajectory.ts
-
-
-class TestPDBQT(BaseTimestepInterfaceTest):
-    __test__ = True
-
-    def setUp(self):
-        u = self.u = mda.Universe(PDBQT_input)
-        self.ts = u.trajectory.ts
-
-
-class TestPQR(BaseTimestepInterfaceTest):
-    __test__ = True
-
-    def setUp(self):
-        u = self.u = mda.Universe(PQR)
-        self.ts = u.trajectory.ts
-
-
-class TestTRJ(BaseTimestepInterfaceTest):
-    __test__ = True
-
-    def setUp(self):
-        u = self.u = mda.Universe(PRM, TRJ)
-        self.ts = u.trajectory.ts
-
-
-class TestNCDF(BaseTimestepInterfaceTest):
-    __test__ = True
-
-    def setUp(self):
+    @pytest.fixture()
+    def universe(self):
         pytest.importorskip('netCDF4')
```

Comment:
I don't think that we need to check for netCDF anymore. We should *always* be able to read ncdf files because we use scipy.io.netcdf for reading.

Reply:
Does this mean that `pytest.importorskip('netCDF4')` shouldn't be present anywhere in the tests as we don't use it anymore?

Reply:
With `git grep` I only found it in `MDAnalysisTests/coordinates/test_xdr.py` and there it is also _not_ needed anymore. We should always be able to read NCDF files.

Reply:
(I must have overlooked these instances when I worked on #506 so thanks for fixing them as you go along.)

Reply:
The other PR where this should be fixed is #1655.